files/es/web/exslt/exsl/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Common (exsl)
+slug: Web/EXSLT/exsl
+translation_of: Web/EXSLT/exsl
+---
+{{XSLTRef}}{{QuickLinksWithSubpages("/es/docs/Web/EXSLT")}}
+
+El paquete EXSLT Common proporciona funciones básicas que amplían las capacidades
+de XSLT. El espacio de nombres para el paquete común es `http://exslt.org/common`.
+
+{{SubpagesWithSummaries}}
```

Comment:
pude ver pequeñas diferencias en el documento pero es por esta macro, creo

Reply:
Noté esto después de hacer _merge_, pero creo que en el párrafo final en lugar de "común" sería "common" hablando del nombre del paquete :$